examples/reference/menus/SplitButton.ipynb

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
When I click the `SplitButton` in the jupyter lab notebook the notebook page jumps around.


https://github.com/user-attachments/assets/2d20b194-5da3-460c-9c7a-98ebd5146230